src/components/BRadio/BRadio.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <div class="container" :class="{ disabled: disabled }">
+    <div role="radiogroup" :aria-labelledby="groupId">
+      <label :id="groupId" class="description">
+      {{ groupLabel }}<span v-if="required" aria-hidden="true">*</span>
+      </label>
+      <div class="button-group" role="radiogroup">
+        <template v-for="(radio, index) in radios">
+          <div class="button" :key="index">
```

Comment:
Shouldn't the :key be on the input here, and not the div?

Reply:
I thought it was supposed to be on every element in a v-for?  Ill research some more.

Reply:
The example here shows putting it on the element you have a v-for on.  I am updating it now.

https://vuejs.org/v2/guide/list.html#key